src/sentry/static/sentry/app/types/iconProps.ts

```diff
@@ -0,0 +1,8 @@
+export interface IconProps extends React.SVGAttributes<SVGElement> {
+  color?: string;
+  size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
Should we try to use the same(-ish) values that we use for `<Button>`?

Reply:
Actually, I was hoping to update button sizes to these at some point. The current sizes for buttons are odd. Reminder - those are 'zero' | 'micro' | 'small' | 'xsmall' | 'xxsmall' | 'large'